docs/pages/workflow/upgrading-expo-sdk-walkthrough.mdx

```diff
@@ -79,16 +79,18 @@ If you manage your own native projects, you will need to manually apply any chan
 
 Each SDK announcement blog post contains deprecations, breaking changes, and anything else that might be unique to that particular SDK version. When upgrading, be sure to check these out to make sure you don't miss anything.
 
+### SDK 52
+
+[Blog Post](https://expo.dev/changelog/2024/11-12-sdk-52)
+
 ### SDK 51
 
 [Blog post](https://expo.dev/changelog/2024/05-07-sdk-51)
 
-### SDK 50
+#### SDK 50
```

Comment:
Was the change intentional?

```suggestion
### Deprecated SDK Version Changelogs

#### SDK 50
```

Reply:
Yes, the last SDK version is deprecated in every SDK release cycle.

Reply:
No, I was referring to removing the heading “Deprecated SDK Version Changelogs” 😄

Reply:
Ah yes, my bad, sorry. You are right, the heading wasn't supposed to be removed. I'll add it back.